### PR TITLE
TST: add test of normed histogram with unequal bins

### PR DESCRIPTION
**Pull request by jakevdp**
Add test for #9557

### CODE REVIEW

**Comment by jakevdp** [comment]
This should be back-ported to 2.1.x

File: lib/matplotlib/tests/test_axes.py [hunk 1]
```diff
@@ -1565,6 +1565,17 @@ def test_hist_step_log_bottom():
     ax.set_ylim(9e-3, 1e3)
 
 
+def test_hist_unequal_bins_normed():
+    # Test correct behavior of normalized histogram with unequal bins
+    # https://github.com/matplotlib/matplotlib/issues/9557
+    rng = np.random.RandomState(57483)
+    t = rng.randn(100)
+    bins = [-3, -1, -0.5, 0, 1, 5]
+    mpl_heights, _, _ = plt.hist(t, bins=bins, normed=True)
```
Comment (jklymak) [inline lib/matplotlib/tests/test_axes.py]: `normed` is deprecated!

Comment (phobson) [reply lib/matplotlib/tests/test_axes.py]: D'oh! Conflated things. Sorry.

Comment (jklymak) [reply lib/matplotlib/tests/test_axes.py]: Its very confusing!

Comment (jakevdp) [reply lib/matplotlib/tests/test_axes.py]: Done

**Comment by jklymak** [comment]
Looks good unless the tests bork...

**Comment by tacaswell** [comment]
@meeseeksdev backport to 2.1.x

**Comment by QuLogic** [comment]
`v2.1.x`?

**Comment by tacaswell** [comment]
@meeseeksdev backport to v2.1.x

**Pull request merged**